client/app/bundles/course/assessment/components/AssessmentForm/translations.ts

```diff
@@ -155,6 +155,12 @@ const translations = defineMessages({
     defaultMessage:
       'When enabled, each submission will have its own timer and will automatically be finalised when its timer ends.',
   },
+  changingTimeLimitWarning: {
```

Comment:
Where is this used?

Reply:
we previously had the discussion of giving the warning to user if they want to change the time limit while attempting submissions already exist, but at the end decided not to go for that.

Hence, this translation is ended up being unused, and I have removed it